core/lib/workarea/warnings.rb

```diff
@@ -86,6 +87,22 @@ def check_auto_capture
     no_shipping: 'purchase!'
   }
 
+**************************************************
+        eos
+      end
+    end
+
+    def check_libvips_version
+      if Configuration::ImageProcessing.libvips_version.present? &&
+          !Configuration::ImageProcessing.libvips?
+
+        warn <<~eos
+**************************************************
+⛔️ WARNING: libvips is installed but will not be used.
```

Comment:
Can we print the version here? Also, I think this would happen if you for some reason had libvips installed but `vips` wasn't in your `$PATH`. Might want to mention that's the case before people spend an hour recompiling something they didn't need to...

```suggestion
⛔️ WARNING: libvips (#{Configuration::ImageProcessing.libvips_version}) is installed in your $PATH but will not be used.
```